docs/README.md

```diff
@@ -74,6 +74,12 @@ The URLs in this message tell you where to find the results of the installation:
 
 -  The **load generator** URL takes you to an interface for generating synthetic traffic to Hipster Shop.
 
+### Recovering from session timeout
+Should your Cloud Shell session timeout due to user inactivity, you will need to launch the custom Cloud Shell image to access the `sandboxctl` command.
+Click the button below to restart the custom Cloud Shell.
+
+[![Restart your Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/cloud-ops-sandbox.git&cloudshell_git_branch=v0.4.0&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image/uncertified:v0.4.0&cloudshell_tutorial=docs/tutorial.md)
```

Comment:
The problem with this is the version numbers have to be updated after each release. We could add these files to our make_release.sh script to make sure it's updated each time, but it's probably best to limit the number of links to update.

Maybe we should point people to cloud-ops-sandbox.dev or something instead?